### DISCUSSION TITLE
Installation guide should explain pros/cons to brew cask install

### DISCUSSION THREAD

**Discussion by mulka** [opensteno/plover#1223] [Q&A and Support]
## Summary

I have homebrew, but not sure what the pros and cons would be of using that over using a download from GitHub. Like, for example, what version of the app would I get if I installed via homebrew? Are there any benefits of using homebrew vs just installing the app via GitHub?

## System

Mac

**Comment by gdwaynewarner** [comment]
I installed Homebrew sometime back, and it downloaded the release version of Plover vice anything new. Deleted that, then got rid of Homebrew.

**Comment by mulka** [comment]
Would love to get any insight.

**Comment by marnen** [comment]
I use Homebrew for dev tools, but downloaded Plover straight from GitHub and installed it like any other application. In general Cask isn’t really that good for my setup anyway, though I’ve used it for a few things; I’ve come to believe that it’s best not to use a package manager for native graphical Mac .apps.

**Comment by user202729** [comment]
@benoit-pierre Why don't you just migrate the issue to a discussion?

**Reply by panathea** [reply]
Thanks for the suggestion. Migrated.

**Comment by panathea** [comment]
We don't manage the brew scripts so support for issues related to them would be outside the scope of this repo. I'd just recommend using the GitHub releases if you are comfortable with that workflow.
